kubernetes/config/kube_config.go

```diff
@@ -0,0 +1,281 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/ghodss/yaml"
+
+	"k8s.io/go/kubernetes/client"
+	"k8s.io/go/kubernetes/config/api"
+)
+
+const (
+	defaultUserAgent          = "Swagger-Codegen/0.1.0a1/go"
```

Comment:
again, probably it should be updated.

Reply:
Yes I think it should be generated. Probably in a release automation as a future step